Mathlib/LinearAlgebra/FreeModule/StrongRankCondition.lean

```diff
@@ -19,46 +19,52 @@ module is well defined.
 
 ## Main result
 
+* `commRing_orzechProperty R` : `R` has the `OrzechProperty`, that is, for any finitely generated
+  `R`-module `M`, any surjective homomorphism `f : N → M` from a submodule `N` of `M` to `M`
+  is injective.
 * `commRing_strongRankCondition R` : `R` has the `StrongRankCondition`.
 
 ## References
 
-We follow the proof given in https://mathoverflow.net/a/47846/7845.
+The original proof was adapted from <https://mathoverflow.net/a/47846/7845>.
```

Comment:
I am not sure it is helpful to refer to the `original` and `new` proof. Because the only thing visible in the file is the new proof. If you think it is good to keep this comment, please indicate for the user where in the git history they can find a formalization of the original proof.

Reply:
I'm fine with removing the old text from the docstring.

Reply:
OK. I have removed the references to old proofs.